### DISCUSSION TITLE
[Feature Request] Elastic Search support for  Percentile Rank aggregation

### DISCUSSION THREAD

**Discussion by mdan1eli** [grafana/grafana#56260] [Observability]
Would be nice to have percentile_rank support has many SLA related metrics could be visualized as single stats panel and colored accordingly to thresholds.

**Comment by winmillwill** [comment]
I don't know that this request is specific to a data source. The single stat panel just uses the TimeSeries object which has a few reducing functions, regardless of data source. A cursory google search turns up the jstat package. It would be nice if we could just include that or something similar and update the editor to list the available functions. I don't know enough about Typescript to know what the convention and level of effort is for calling un-typed javascript.

I'm happy to give it a try with a bit of guidance.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
